connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MsSqlServerIntegrationSuite.scala

```diff
@@ -265,7 +273,7 @@ class MsSqlServerIntegrationSuite extends DockerJDBCIntegrationSuite {
     assert(types(7).equals("class java.lang.String"))
     assert(types(8).equals("class [B"))
     assert(row.getString(0).length == 10)
-    assert(row.getString(0).trim.equals("the"))
+    assert(row.getString(0).equals("the".padTo(10, ' ')))
```

Comment:
This seems to be ANSI result. Is this safe in non-ANSI CI, @yaooqinn ?
- https://github.com/apache/spark/actions/workflows/build_non_ansi.yml

Reply:
Char padding is ANSI-irrelevant, AFAIK

Reply:
Anyway, I removed this LOC

Reply:
Thanks. Yes, that would be safe.